elide-annotations/src/main/java/com/yahoo/elide/annotation/OnCreatePostCommit.java

```diff
@@ -21,4 +21,12 @@
 @Target({ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface OnCreatePostCommit {
+    /**
+     * Field name on which the annotated method is only triggered if that field is modified.
+     * If value is empty string, then trigger once when the object is created.
+     * If value is "*", then trigger for all field modifications.
```

Comment:
Do you mean `If the value is "*", then this method will be triggered once for each field that the user sent in the creation request`?

Same question for the other annotations.